src/onboarding/steps.ts

```diff
@@ -82,9 +81,7 @@ export const onboardingPropsSelector = createSelector(
       StatsigFeatureGates.SHOW_CLOUD_ACCOUNT_BACKUP_RESTORE
     )
 
-    const { skipVerification } = getExperimentParams(
-      ExperimentConfigs[StatsigExperiments.ONBOARDING_PHONE_VERIFICATION]
-    )
+    const skipVerification = getFeatureGate(StatsigFeatureGates.SHOW_ONBOARDING_PHONE_VERIFICATION)
```

Comment:
should this be `const skipVerification = !getFeatureGate(StatsigFeatureGates.SHOW_ONBOARDING_PHONE_VERIFICATION)`? It looks like the feature gate is a boolean saying to show verification but then this const is a boolean saying to skip it.

Reply:
Fixed in 780994ed7e9a7f1566429cdea1a7f15809717bf0! Interestingly, the tests missed this, but it appears expected due to how they set up mock props.